### PR TITLE
Various improvements to ENCODE metadata

### PR DESCRIPTION
**Pull request by danxmoran**
Hopefully not too much fever-code...

Simple changes:
1. Add `url` fields to both file and donor rows
2. Add `age_units` column to donors

More complicated:
1. For files tied to multiple experiments, strip out all fields tied to the control experiments (vs. just the target-level fields).
2. Get file-level audit info by querying file entities directly, instead of querying experiments and trying to extract out the file-specific information

### CODE REVIEW

File: encode-ingest/src/main/scala/org/broadinstitute/gdr/encode/EncodeFields.scala [hunk 1]
```diff
@@ -3,20 +3,13 @@ package org.broadinstitute.gdr.encode
 /** Bucket for field names used while transforming raw metadata pulled from ENCODE. */
 object EncodeFields {
 
-  ///////////////////////////////////////////////////
```
Comment (danxmoran) [inline encode-ingest/src/main/scala/org/broadinstitute/gdr/encode/EncodeFields.scala]: I wasn't a huge fan of this big-bag-of-field-names approach, so started to peel constants back into the specific steps that need them.

File: encode-ingest/src/main/scala/org/broadinstitute/gdr/encode/clp/Encode.scala [hunk 2]
```diff
@@ -48,34 +47,6 @@ object Encode
             .map(new PrepareIngest(_))
         }
 
-        val genFirecloud = Opts.subcommand(
```
Comment (danxmoran) [inline encode-ingest/src/main/scala/org/broadinstitute/gdr/encode/clp/Encode.scala]: The TSV-generating code got in the way of the other refactoring in here, and I've had plans to replace it with equivalent JSON-generating code, so I just scrapped it for now.

Will make a ticket for the JSON generation.

File: encode-ingest/src/main/scala/org/broadinstitute/gdr/encode/steps/PrepareIngest.scala [hunk 3]
```diff
@@ -5,7 +5,7 @@ import cats.effect.Effect
 import cats.implicits._
 import fs2.{Scheduler, Stream}
 import org.broadinstitute.gdr.encode.steps.download._
-import org.broadinstitute.gdr.encode.steps.google.BuildStsManifest
+//import org.broadinstitute.gdr.encode.steps.google.BuildStsManifest
```
Comment (RoriCremer) [inline encode-ingest/src/main/scala/org/broadinstitute/gdr/encode/steps/PrepareIngest.scala]: why not just delete?

Comment (danxmoran) [reply encode-ingest/src/main/scala/org/broadinstitute/gdr/encode/steps/PrepareIngest.scala]: Was a temp comment for testing (scalac complains on unused imports), should be reverted in the latest commit.

**Pull request merged**